index.bs

```diff
@@ -980,15 +985,25 @@ partial interface HTMLIFrameElement {
     policy</a>, which may be empty.
     1. If |element| is not an <{iframe}> element, then return an empty [=policy
        directive=].
+    1. Let |allowAttribute| be the value of |element|'s <{iframe/allow}>
+       attribute.
     1. Let |container policy| be the result of running <a abstract-op>Parse policy
-      directive</a> given the value of |element|'s <{iframe/allow}> attribute,
-      the [=Document/origin=] of |element|'s [=node document=], and |element|'s
-      <a>declared origin</a>.
+      directive</a> given |allowAttribute|, the [=Document/origin=] of |element|'s
+      [=node document=], and |element|'s <a>declared origin</a>.
     1. If |element|'s <{iframe/allowfullscreen}> attribute is specified, and
        |container policy| does not [=map/contain=] an entry for the
        <code>fullscreen</code> [=policy-controlled feature|feature=].
       1. [=map/Set=] |container policy|[<code>fullscreen</code>] = <a>the
          special value <code>*</code></a>.
+    1. [=map/For each=] |feature| in |container policy|:
+      1. If [=feature=]'s <a>allowlist</a> does not [=list/contain=] <a>the special
+         value <code>*</code></a>, then [=list/for each=] |origin| in
```

Comment:
This might not acutally work, now that we have [Wildcards in Permissions Policy Origins](https://github.com/w3c/webappsec-permissions-policy/pull/516), the allowlist doesn't necessarily contain origins.
Instead, if it's not `*`, it contains [permissions-source-expressions](https://w3c.github.io/webappsec-permissions-policy/#permissions-source-expression) which could be concrete origins, or could include wildcards, like `https://*.github.io` or even `https://*`.
I'm not sure how you'd want to handle that; maybe constructing a "representative origin" for such a pattern, and testing *that*, or ignoring patterns with wildcards (or something else?)